src/stories/pages/Logout/Logout.js

```diff
@@ -0,0 +1,67 @@
+import React, { useContext } from 'react';
+import { useHistory } from 'react-router-dom';
+import { AuthContext } from '../../../contexts/auth-provider';
+import { HorizontalTitle } from
+  '../../components/HorizontalTitle/HorizontalTitle';
+import { Constrain } from '../../layouts/Constrain/Constrain';
+import { Button } from '../../components/Button/Button';
+import { logOut } from '../../../firebase/api/users';
+/**
+ * Component for logout page.
+ *
+ * @component
+ * @return {object} (
+ *   <Logout>{children}</Logout>
+ * )
+ */
+export const Logout = () => {
+  const history = useHistory();
+
+  const { resetUserData } = useContext(AuthContext);
+
+
+  const logUserOut = (e) => {
+    e.preventDefault();
+
+    logOut().then((response) => {
+      if (response.status === 200) {
+        // REDIRECT TO LOGIN
+        // ERROR HANDLING
+        resetUserData();
+        history.push('/login');
+      } else {
+        alert(response.status);
+      }
+    });
+  };
+
+  const cancelLogout = (e) => {
+    history.push('/dashboard');
+  };
+  return (
+    <Constrain>
+      <HorizontalTitle
+        modifierClasses="horizontal-title--medium"
+        eyebrow="Logout"
+        title="You sure?"
+        content={<>
+          <Button
+            modifierClasses={'button--small button--secondary'}
+            disabled={true}
+            isButton={true}
+            text="Yes"
+            onClick={(e) => logUserOut(e)}
+          />
```

Comment:
Design subjective preference: Change the content of the HorizontalTitle to the string `"Are you sure you want to log out?"` and move the 'Yes' and 'No' buttons outside at the same level as the HorizontalTitle (but inside the Constrain)?